Lib/glyphsLib/builder/user_data.py

```diff
@@ -85,6 +103,8 @@ def to_ufo_layer_lib(self, master, ufo, ufo_layer):
 def to_ufo_layer_user_data(self, ufo_glyph, layer):
     user_data = layer.userData
     for key in user_data.keys():
+        if self.minimal and not key.startswith(GLYPHS_MATH_PREFIX):
+            continue
```

Comment:
same here, I don't understand why you skip exporting layer.userData if self.minimal is False